src/cmap/test/mod.rs

```diff
@@ -21,12 +21,15 @@ use crate::{
     test::{
         assert_matches,
         run_spec_test,
+        CmapEvent,
```

Comment:
This looks like a holdover from the test that was removed

Reply:
Oh whoops, fixed.